ui/src/shared/components/Notification.js

```diff
@@ -50,6 +52,14 @@ class Notification extends Component {
     )
   }
 
+  onNotificationRef = ref => {
+    if (ref) {
+      const {height} = ref.getBoundingClientRect()
+      this.setState({height})
+      this.notificationRef = ref
```

Comment:
what if you put this at the top of this if block, and then use `this.notificationRef.getBoundClientRect()` etc.? and then you can factor this and lines 30-31 out into a function for setting the height? or if not factoring it out, at least they'll be consistent syntax

Reply:
👍